libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -140,23 +153,17 @@ export const NavbarIconButtonWrapper = styled('div')`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: ${({theme}) => theme.spacing(6.5)};
-  height: ${({theme}) => theme.spacing(6.5)};
+  width: var(--navbar-height);
+  height: var(--navbar-height);
```

Comment:
könnte einfach height: 100% machen und aspect-ratio: 1 oder?

Reply:
Der NavbarIconWrapper bestimmt die Grösse des ganzen Headers, ist jedoch nicht direkt im NavbarWrapper enthalten, sondern noch in einige weitere Elemente eingepackt, die keine explizite Höhe haben. Deshalb funktioniert `height: 100%` nicht. Aber ich habe dafür die explizite Höhe des NavbarWrappers wieder weggenommen, weil die auch keinen Einfluss hat.